fetch.bs

```diff
@@ -4367,9 +4384,60 @@ steps:
   <p>Let <var>processResponseEndOfBody</var> be the following steps:
 
   <ol>
+   <li><p>Let <var>unsafeEndTime</var> be the <a>unsafe shared current time</a>.
+
    <li><p>Set <var>fetchParams</var>'s <a for="fetch params">request</a>'s
    <a for=request>done flag</a>.
 
+   <li>
+    <p>If <var>fetchParams</var>'s <a for="fetch params">request</a>'s
+    <a for=request>destination</a> is "<code>document</code>", then set <var>fetchParams</var>'s
+    <a for="fetch params">controller</a>'s <a for="fetch controller">full timing info</a> to
+    <var>fetchParams</var>'s <a for="fetch params">timing info</a>.
+
+    <p>Set <var>fetchParams</var>'s <a for="fetch params">controller</a>'s <a
+    for="fetch controller">report timing steps</a> to the following steps given a <a
+    for=/>global object</a> <var>global</var>:
+
+    <ol>
+     <li><p>If <var>fetchParams</var>'s <a for="fetch params">request</a>'s <a for=request>URL</a>'s
```

Comment:
Do we want to use the scheme name ("https" vs. "http")? Or should we check if the URL is trustworthy?

Reply:
http(s). It's to prevent creating resource timing entries for e.g. `data` or `blob` URLs